README.md

```diff
@@ -39,9 +39,9 @@ You can decode the base64 encoded basicAuth information using the `scripts/get_c
 
 ## Development
 
--   Init your pipenv environment (`pipenv install --dev`)
+-   Init your pipenv environment (`poetry install`)
```

Comment:
Should be "Init you virtual environment" or "python environment" or "poetry environment", now.